packages/clarity-js/src/data/variable.ts

```diff
@@ -0,0 +1,35 @@
+import { Constant, Event, VariableData } from "@clarity-types/data";
+import encode from "./encode";
+
+export let data: VariableData = null;
+
+export function set(variable: string, value: string): void {
+    log(variable, value);
+    encode(Event.Variable);
+}
+
+export function identify(userId: string, sessionId: string = null, pageId: string = null): void {
+    log(Constant.RESERVED_USER_ID_VARIABLE, userId);
```

Comment:
Do we care if the user sets these directly instead of calling this?

Reply:
No. Not worth adding extra code for that in my opinion. Whenever we work on the documentation, this can be included there.